gui/guihtml/src/TGHtmlParse.cxx

```diff
@@ -1103,8 +1102,7 @@ void TGHtml::TokenizerAppend(const char *text)
       return;
    }
 
-   // coverity[secure_coding]
-   strcpy(&fZText[fNText], text);
+   strlcpy(&fZText[fNText], text, fNAlloc);
```

Comment:
Here should be:
```
strlcpy(&fZText[fNText], text, fNAlloc - fNText);
```
Mean you only can owerwrite so many bites in target

Reply:
Well, OK, thanks